### PR TITLE
Issue#1728 film page layout

### LINKED ISSUE #1666: update path for uploading images & files
path with movieId

### LINKED ISSUE #1728: Film page layout
## UI
[Zeplin - Marketplace activity tab](https://app.zeplin.io/project/5ca741b3ed21e79faa266664/screen/5e26d365922d4d961b4b0435)
[Zeplin - Film details tab ](https://app.zeplin.io/project/5ca741b3ed21e79faa266664/screen/5e26d4eae6a7365468f53e91)
[Zeplin - Sales tab](https://app.zeplin.io/project/5ca741b3ed21e79faa266664/screen/5e26d5fc683ce156ce01990b)

## Acceptance criteria
- Create Catalog Movie Detail component #1449 
- Use what @GrandSchtroumpf has done so far here http://localhost:4200/c/o/dashboard/titles/0
- This issue is about the top part of the page:
- [x] Movie poster
- [x] Movie type
- [x] Movie status
- [x] Movie runtime 
- [x] Movie Title
- [x] Movie Director
- [x] Movie origin countries
- [x] Movie prodution year
- [x] View Sales Report dropdown button
- [x] Tabs Marketplace Activity / Film Detail / Sales linked to routing

### PR DESCRIPTION
**Pull request by jenna1k**
resolve #1728 
resolve #1666

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts [hunk 1]
```diff
@@ -16,9 +25,18 @@ export class TitleViewComponent implements OnInit {
     path: 'avails',
     label: 'Avails'
   }];
-  constructor() { }
+  constructor(
+    private movieQuery: MovieQuery,
+    private orgQuery: OrganizationQuery,
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts]: It seems that you don't need that here

File: libs/ui/src/lib/media/cropper/cropper/cropper.component.ts [hunk 2]
```diff
@@ -142,8 +141,7 @@ export class CropperComponent implements ControlValueAccessor {
         throw new Error('No image cropped yet');
       }
       this.nextStep('upload');
-      const fileName = sanitizeFileName(this.file.name);
-      this.ref = this.storage.ref(`${this.folder}/${fileName}`);
+      this.ref = this.storage.ref(`${this.folder}/${this.file.name}`);
```
Comment (phpgeekfr) [inline libs/ui/src/lib/media/cropper/cropper/cropper.component.ts]: why did you removed `sanitizeFileName` ? What happens if we upload 2 different files with the same name ?

File: apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts [hunk 3]
```diff
@@ -1,11 +1,20 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { MovieQuery, Movie } from '@blockframes/movie';
+import { OrganizationQuery } from '@blockframes/organization';
+import { Observable } from 'rxjs';
+import { getLabelByCode } from '@blockframes/utils/static-model/staticModels';
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts]: This method have been renamed to `getLabelBySlug` . You did not had errors when doing ng-serve ?

File: apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts (hunk 1 above)
Comment (jenna1k) [reply apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts]: removed

File: apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts (hunk 3 above)
Comment (jenna1k) [reply apps/catalog/catalog/src/app/dashboard/title/view/view.component.ts]: changed

File: libs/ui/src/lib/media/cropper/cropper/cropper.component.ts (hunk 2 above)
Comment (jenna1k) [reply libs/ui/src/lib/media/cropper/cropper/cropper.component.ts]: we need to display image name in the movie-tunnel summary page
for duplicated file name, maybe I can add Date() on the file name

Comment (phpgeekfr) [reply libs/ui/src/lib/media/cropper/cropper/cropper.component.ts]: I'm currently working on this issue : https://github.com/blockframes/blockframes/issues/1665
To add the originalFileName.

**Pull request merged**